download-singularity-img.nf

```diff
@@ -32,7 +32,7 @@ def helpMessage() {
     Mandatory arguments:
       --outdir [path]               Output directory for downloading
       -profile [str]                Configuration profile to use. Can use multiple (comma separated)
-                                    Available: standard, conda, docker, singularity, awsbatch, test
+                                    Available: standard, docker, singularity, awsbatch, test
```

Comment:
I think we can remove awsbatch as well from the comments, no one is going to use singularity on AWS